lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -508,6 +504,19 @@ def create_tmpdir_on(host, path_prefix = '', user=nil)
           end
         end
 
+        # 'echo' the provided value on the given host
+        # @param [Host] host The host to execute the 'echo' on
+        # @param [String] val The string to 'echo' on the host
+        # @return [String] The echo'ed value
+        def echo_on_host host, val
```

Comment:
That is a chunk of code moved from host_prebuilt_steps to its new home in host_helpers.  I'm not keen on altering it as a ride along, but it should be noted and filed as a separate issue.